src/app/hooks/useTransactions.ts

```diff
@@ -17,16 +18,38 @@ export const useTransactions = () => {
           accountId,
           limit,
         });
+        const fetchInvoice = await api.getInvoices({ isSettled: true, limit });
```

Comment:
it would be nice to use SWR, there's a delay each time you go back to the home page right now. Maybe that could be done in a follow up PR?

Reply:
let's decide this along with a new method that returns both incoming and outgoing transactions. we also need to decide it on other connectors as well. i will probably prefer to do it as follow up pr